modules/nodes-descheduler-profiles.adoc

```diff
@@ -90,9 +90,70 @@ Do not enable `CompactAndScale` with any of the following profiles: `LifecycleAn
+Use the `DevKubeVirtRelieveAndMigrate` or `LongLifecycle` profile to enable the descheduler on a virtual machine.
+
+[IMPORTANT]
+====
+You can not have both `DevKubeVirtRelieveAndMigrate` and `LongLifeCycle` enabled at the same time.
+====
 endif::virt[]
 
+`DevKubeVirtRelieveAndMigrate`:: This profile is an enhanced version of the `LongLifeCycle` profile. It evicts pods from high-cost nodes to reduce overall resource expenses and enable workload migration. It also periodically rebalances workloads to help maintain similar spare capacity across nodes, which supports better handling of sudden workload spikes. Node costs can include the following:
+
++
+--
+* **Resource utilization**: Increased resource pressure raises the overhead for running applications.
+* **Node maintenance**: A higher number of containers on a node increases resource consumption and maintenance costs.
+
+The profile enables the `LowNodeUtilization` strategy with the `EvictionsInBackground` alpha feature. The profile also exposes the following customization fields:
+
+* `devActualUtilizationProfile`: Enables load-aware descheduling.
+* `devLowNodeUtilizationThresholds`: Sets experimental thresholds for the `LowNodeUtilization` strategy. This field cannot be used together with `devDeviationThresholds`.
+* `devDeviationThresholds`: Treats nodes with below-average resource usage as underutilized to help redistribute workloads from overutilized nodes. This field cannot be used together with `devLowNodeUtilizationThresholds`. Supported values: `Low` (10%:10%), `Medium` (20%:20%), `High` (30%:30%), `AsymmetricLow` (0%:10%), `AsymmetricMedium` (0%:20%), `AsymmetricHigh` (0%:30%).
+* `devEnableSoftTainter`: Enables the soft-tainting component to dynamically apply or remove soft taints as scheduling hints.
+
+.Example configuration:
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1
+kind: KubeDescheduler
+metadata:
+  name: cluster
+  namespace: openshift-kube-descheduler-operator
+spec:
+  managementState: Managed
+  deschedulingIntervalSeconds: 30
+  mode: "Automatic"
+  profiles:
+    - DevKubeVirtRelieveAndMigrate
+  profileCustomizations:
+    devEnableSoftTainter: true
+    devDeviationThresholds: AsymmetricLow
+    devActualUtilizationProfile: PrometheusCPUCombined
+----
+
+The `DevKubeVirtRelieveAndMigrate` profile requires PSI metrics to be enabled on all worker nodes. You can enable this by applying the following `MachineConfig` custom resource:
+
+.Example `MachineConfig`
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker
+  name: 99-openshift-machineconfig-worker-psi-karg
+spec:
+  kernelArguments:
+    - psi=1
+----
+
+You can use this profile with the `SoftTopologyAndDuplicates` profile to also rebalance pods based on soft topology constraints which can be useful in Hosted Control Plane (HCP) environments.
```

Comment:
```suggestion
You can use this profile with the `SoftTopologyAndDuplicates` profile to also rebalance pods based on soft topology constraints which can be useful in Hosted Control Plane (HCP) environments.
```

The attributes file has hosted control planes as lowercase:

:hcp-capital: Hosted control planes
:hcp: hosted control planes